### ISSUE TITLE
Role sync not working correctly

### ISSUE THREAD

**Issue by joepio** [Maximophone/ProjectBot#2]
The bot is supposed to synchronize discord roles to the Airtable (not the other way around), but I think this is not working correctly, since there are quite a few people missing in the airtable that have been added in Discord. 

For example, there are 41 people with the `organizer` role in discord, but only 6 members in the airtable team.

**Comment by Maximophone** [comment]
It is not synchronising in this direction at the moment, only updating someone's role in airtable when we change the role in discord

**Comment by joepio** [comment]
>  in this direction

I think I mean the same direction: discord to airtable.

Does it only update when the _role is changed_ in discord?

**Comment by Maximophone** [comment]
Yes, that's the current behaviour. Adding synchronization the other way should not be too hard

**Comment by joepio** [comment]
I think we're miscommunicating: I'm not asking for update in a different direction (e.g. from Airtable to Discord, I don't think we should do that direction at all), but I'd like the discord to airtable sync to work reliably, so that values in the discord modify the airtable values. That means the airtable should be slave - all edits there should be discarded IMO. The bot should periodically read the values from discord and update the airtable accordingly.

What do you think?

**Comment by Maximophone** [comment]
Ah, I see, I thought about this before, and I was not very comfortable with discord being the source of truth and airtable being a mirror. I will try to remember my reasons here:
- We will often want to make changes to airtable directly
- I learnt a lesson from the onboarding process. Originally, the emoji reactions were the source of truth about onboarding, that was reflected in the database. But this was really cumbersome and prone to problems because discord is not a database. This also makes us dependent on discord's features and at risk if they decide to change them
- Learning from that experience, what could go wrong? It is extremely unlikely that discord changes the roles system so I don't think that's a real risk. There are unknown unknowns
 
As I am writing these reasons down, I do not feel strongly opposed anymore to discord being the source of truth for team roles. My main objection is that it will make it impossible to update a role from airtable directly
